batch/batch/server/server.py

```diff
@@ -80,29 +109,18 @@ def jsonify(data):
 
 class JobTask:  # pylint: disable=R0903
     @staticmethod
-    def copy_task(job_id, task_name, files, copy_service_account_name):
+    def copy_task(job_id, task_name, files):
         if files is not None:
-            assert copy_service_account_name is not None
-            authenticate = 'gcloud -q auth activate-service-account --key-file=/gcp-sa-key/key.json'
-            copies = ' & '.join([f'gsutil cp {src} {dst}' for (src, dst) in files])
-            wait = 'wait'
-            sh_expression = f'{authenticate} && ({copies} ; {wait})'
+            authenticate = 'set -ex; gcloud -q auth activate-service-account --key-file=/gsa-key/key.json'
+            copies = ' && '.join([f'gsutil cp {src} {dst}' for (src, dst) in files])
```

Comment:
Did you mean to make this a serial copy?

Reply:
yeah, i couldn't figure out an easy way to check if a background task had failed in batch. this seems simpler